### PR TITLE
add some comments to clarify command-line argument munging in #482

### PR DESCRIPTION
**Pull request by apoelstra**
In #482 there was some question about what my argument-handling logic was doing. This adds comments to explain it.

### CODE REVIEW

File: src/bin/cargo-miri.rs [hunk 1]
```diff
@@ -109,6 +110,8 @@ fn main() {
                     }
                 }
                 (true, "lib") => {
+                    // For libraries we call `cargo rustc -- --test <rustc args>`
+                    // Notice now that `--test` is a rustc arg rather than a cargo arg.
```
Comment (RalfJung) [inline src/bin/cargo-miri.rs]: "libraries" here means "`#[test]` functions", right?

Comment (apoelstra) [reply src/bin/cargo-miri.rs]: Yes.

Comment (RalfJung) [reply src/bin/cargo-miri.rs]: Okay. Could you add that?

Took me a while to figure that out (originally, as in some months ago, I thought these are also just test binaries for cargo, I didn't know where the magic lived).

**Comment by RalfJung** [comment]
Thanks!

**Pull request merged**